trio/_core/_run.py

```diff
@@ -47,14 +51,16 @@
 if sys.version_info < (3, 11):
     from exceptiongroup import BaseExceptionGroup
 
-DEADLINE_HEAP_MIN_PRUNE_THRESHOLD = 1000
+T = TypeVar("T")
```

Comment:
Should we try going for a more descriptive `TypeVar`? `Fn = TypeVar("Fn", bound=Callable[..., Any])`

Completely optional though, what we have now works.

Reply:
I went with a more descriptive type var.